docs/book/component-gallery/orchestrators/orchestrators.md

```diff
@@ -60,6 +60,19 @@ using the orchestrator is as simple as executing a python file which
 python file_that_runs_a_zenml_pipeline.py
 ```
 
+### Inspecting Runs in the Orchestrator UI
+
+If your orchestrator comes with a separate user interface (currently: Kubeflow, 
```

Comment:
```suggestion
If your orchestrator comes with a separate user interface (for example: Kubeflow, 
```

Removed the currently as then this is something we have to maintain + remember to update.